package.json

```diff
@@ -60,7 +60,7 @@
     "lib"
   ],
   "peerDependencies": {
-    "diagram-js": "^11.3.0 || ^12 || ^13.0.0 || ^14.0.0",
+    "diagram-js": "^15.0.0",
```

Comment:
Do we intend to break with the past here? This will require the next `table-js` release to be a major breaking change.

```suggestion
    "diagram-js": "^11.3.0 || ^12 || ^13.0.0 || ^14.0.0 || ^15.0.0",
```

Reply:
Totally, we should not make a breaking change if it's not required.